### DISCUSSION TITLE
Any GUI planned ?

### DISCUSSION THREAD

**Discussion by bitsByB** [tfeldmann/organize#210] [Q&A]
Hi Thomas, thanks for this software. Are you planning to make a GUI for it or are you aware of one? thank you for your time

**Comment by tfeldmann** [comment]
Yes. I'm planning a small web interface built on flask. I'm in the process of refactoring the codebase and cleaning up the interfaces so this becomes possible.

**Reply by Zaldor** [reply]
Hi Thomas, any update on this?

**Reply by tfeldmann** [reply]
The refactoring is done in v3 and a GUI would be possible and pretty much only grunt work. I have a basic GUI where you can start and see some results, but it's far from usable.

The problem with a web app is working with local paths is not very comfortable and the OS's dialogs for folder selection etc are not available so you would have to bring your own file hierarchy explorer. This is much work and not so much fun and I don't have a clear idea how to tackle this at the moment. Also it's a lot of work building the UI for each action and filter. I looked into `react-jsonschema-form` but I would rather write svelte than react.

**Reply by bitsByB** [reply]
I understand, thanks for the new version, will test it soon. thanks @tfeldmann :)

**Comment by Frewchen** [comment]
@tfeldmann Fantastic project! Very useful.
Came across this post and was looking if there's an update on GUI?
